### PR TITLE
Close show message panel on just have warning message,and update tips style.

### PR DESCRIPTION
**Pull request by fishg**
Replace tips style to lightweight style.

### CODE REVIEW

**Comment by fishg** [comment]
qq20150608-2
qq20150608-1

**Comment by fishg** [comment]
@jchannon Please review this pull request.

File: listeners/add_file_to_project.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def on_post_save(self, view):
         if not helpers.is_csharp(view):
             return
 
-        omnisharp.get_response(view, '/addtoproject', self._handle_addtoproject)
```
Comment (jchannon) [inline listeners/add_file_to_project.py]: Why have you commented this out?

File: listeners/syntax.py [hunk 2]
```diff
@@ -64,15 +66,25 @@ def _handle_codeerrors(self, data):
...
+                # print('underlines')
                 self.view.settings().set("oops", oops_map)
-                self.view.add_regions("oops", self.underlines, "illegal", "", sublime.DRAW_NO_FILL + sublime.DRAW_NO_OUTLINE + sublime.DRAW_SQUIGGLY_UNDERLINE)
-                self.view.window().run_command("show_panel", {"panel": "output.variable_get"})
+                self.view.add_regions("oops", self.errlines, "sublimelinter.mark.error", "circle",  sublime.DRAW_NO_FILL|sublime.DRAW_NO_OUTLINE|sublime.DRAW_SOLID_UNDERLINE )
+                if bool(helpers.get_settings(self.view,'omnisharp_onsave_showerrorwindows')):
+                    self.view.window().run_command("show_panel", {"panel": "output.variable_get"})
+            if len(self.warninglines) > 0:
+                # print('underlines')
+                self.view.settings().set("oops", oops_map)
```
Comment (jchannon) [inline listeners/syntax.py]: I think we should still show warnings if  `bool(helpers.get_settings(self.view,'omnisharp_onsave_showerrorwindows'))` is `true`

**Comment by fishg** [comment]
@jchannon 
Comment addtoproject maybe my test modify,I will revert that modify.
I think add a omnisharp_onsave_showwarningwindows for control warning log show maybe better.

File: OmniSharpSublime.sublime-settings [hunk 3]
```diff
@@ -3,4 +3,7 @@
     "omnisharp_auto_start": true,
     "omnisharp_server_config_location":"",
     "omnisharp_onsave_codecheck": true,
-}
\ No newline at end of file
+    "omnisharp_onsave_showwarningwindows":true,
+    "omnisharp_onsave_showerrorwindows":true,
+    "omnisharp_debug_log":false
```
Comment (jchannon) [inline OmniSharpSublime.sublime-settings]: can you remove this, the output is a bit odd plus its useful for people to see the detail when saying there is a problem :smile:

**Comment by jchannon** [comment]
All looks good. Whilst you're in this area do you fancy ordering the output via line number?

For example the output is not in order as it reports errors/warnings randomly ir appears

screen shot 2015-06-13 at 09 51 05

File: OmniSharpSublime.sublime-settings (hunk 3 above)
Comment (fishg) [inline OmniSharpSublime.sublime-settings]: @jchannon Ok.I make that's change for my selft use.I thought if I don't request merge last commit it's not request the commit for merge.

**Comment by jchannon** [comment]
Thanks :+1:

**Pull request merged**